lib/Doctrine/Persistence/ObjectRepository.php

```diff
@@ -34,10 +34,10 @@ public function findAll() : array;
      * an UnexpectedValueException if certain values of the sorting or limiting details are
      * not supported.
      *
-     * @param mixed[]       $criteria
-     * @param string[]|null $orderBy
+     * @param array<string, mixed> $criteria
+     * @param array<string, mixed> $orderBy
```

Comment:
This was `string[]`: new format seems wrong

Reply:
I changed it to `@param array<string, string> $orderBy`